bayeux.go

```diff
@@ -238,39 +246,26 @@ func (b *Bayeux) connect(out chan TriggerEvent) chan TriggerEvent {
 	return out
 }
 
-func GetSalesforceCredentials() Credentials {
-	clientID := mustGetEnv("SALESFORCE_CONSUMER_KEY")
-	clientSecret := mustGetEnv("SALESFORCE_CONSUMER_SECRET")
-	username := mustGetEnv("SALESFORCE_USER")
-	password := mustGetEnv("SALESFORCE_PASSWORD")
-	tokenURL := mustGetEnv("SALESFORCE_TOKEN_URL")
+func GetSalesforceCredentials(ap *AuthenticationParameters) (*Credentials, err) {
```

Comment:
in this case, it's better to use a value instead of a reference. We don't need nil values here.